### ISSUE TITLE
Spring input: consider making the example bar vertical

### ISSUE THREAD

**Issue by marcofugaro** [gsimone/leva#48]
I am talking about the bar on the bottom that shows the example spring:

https://user-images.githubusercontent.com/7217420/106356516-60326f80-6300-11eb-8fc2-411f745e681a.mov

It would make more sense if the bar was vertical and would correspond to the height of the curve. This way it can be easier understood.

Reference: [GSAP Ease visualizer](https://greensock.com/ease-visualizer/)

<img src="https://user-images.githubusercontent.com/7217420/106356554-b8697180-6300-11eb-808e-08345dd09a13.png" width="400" />

**Comment by dbismut** [comment]
True... problem is that it would take a lot of vertical space wouldn't it? Or we should make the controller way taller?

**Comment by marcofugaro** [comment]
No, If it follows the curve, it takes the same vertical space as the curve, here I made a design:

<img src="https://user-images.githubusercontent.com/7217420/106357965-ab04b500-6309-11eb-8f28-fbebc1cce73c.png" width="400" />

**Comment by dbismut** [comment]
I got that, I meant to say that the range of animation would be really small so you wouldn't get a clear grasp of how the spring actually feels. When using the width you do get a better sense of how the spring feels. To get the best of both, we would need to increase the height of the canvas. Could be done though, maybe we can have the spring in a less contrived space.

**Comment by marcofugaro** [comment]
Hmm, maybe it gets better if we add a visual indicator such as the circle in the GSAP Visualizer?

**Comment by dbismut** [comment]
Good idea let's start with this.
